src/lowering/desugar.ml

```diff
@@ -1288,6 +1294,8 @@ let transform_unit_body (u : S.comp_unit_body) : Ir.comp_unit =
       I.ActorU (None, ds, fs, u, t)
     | _ -> assert false
     end
+  | S.FileU str ->
```

Comment:
`str` is meant to be unused?

Reply:
Good point, maybe this code is dead. Will check.